packages/foreman/foreman/foreman.spec

```diff
@@ -792,7 +804,7 @@ find %{buildroot}%{_datadir}/%{name} -type d -name "test" |xargs rm -rf
 rm -f %{buildroot}%{_datadir}/%{name}/bin/spring
 
 # Move config files to %{_sysconfdir}
-mv %{buildroot}%{_datadir}/%{name}/config/database.yml.example %{buildroot}%{_datadir}/%{name}/config/database.yml
+#mv %{buildroot}%{_datadir}/%{name}/config/database.yml %{buildroot}%{_datadir}/%{name}/config/database.yml
```

Comment:
This is IMHO still good for fresh installs to keep

Reply:
This is purely to allow testing this.